bin/juxtaposer/config/config.go

```diff
@@ -0,0 +1,122 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	formatter_api "github.com/cyberark/secretless-broker/bin/juxtaposer/formatter/api"
+)
+
+// Config is the main structure used to define the perfagent parameters
+type Config struct {
+	Backends   map[string]Backend                        `yaml:"backends"`
+	Comparison Comparison                                `yaml:"comparison"`
+	Driver     string                                    `yaml:"driver"`
+	Formatters map[string]formatter_api.FormatterOptions `yaml:"formatters"`
+}
+
+type Backend struct {
+	Database    string `yaml:"database"`
+	Debug       bool   `yaml:"debug"`
+	Description string `yaml:"description"`
+	Host        string `yaml:"host"`
+	Ignore      bool   `yaml:"ignore"`
+	Password    string `yaml:"password"`
+	Port        string `yaml:"port"`
+	SslMode     string `yaml:"sslmode"`
+	Socket      string `yaml:"socket"`
+	Username    string `yaml:"username"`
+}
+
+type Comparison struct {
+	BaselineBackend             string `yaml:"baselineBackend"`
+	BaselineMaxThresholdPercent int    `yaml:"baselineMaxThresholdPercent"`
```

Comment:
Do we need this?  The stats we were doing yesterday -- % of requests over threshold A, B, C, D, etc -- seems to be a superset of this stat.  

Or would that be too big a change at this point?

Reply:
Yeah without this field, none of the data works anymore ☹️ . We'll need to update the tool to perform the same operations that our python/ruby tools do soon to fix this.